templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
+              kube-apiserver-$(hostname -f) \
+              --type=LoadBalancer \
+              --port 443 \
+              --target-port=6443 \
+              --name apiserver-public
+
+            # Wait until it's available
+            set +e
```

Comment:
I think I'm just going to add a `|| true` after the kubectl line and have the break condition be checking if $LB_HOSTNAME is nonempty, that way we can keep the flag as it is.